libethereum/Client.cpp

```diff
@@ -650,6 +651,21 @@ void Client::resyncStateFromChain()
 	}
 }
 
+void Client::resetState()
+{
+	State newPreMine;
+	DEV_READ_GUARDED(x_preMine)
```

Comment:
This was just some wrong indent. The style for two guards on the same line take from resyncStateFromChain above.